src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -1348,17 +1348,24 @@ namespace EvaporativeFluidCoolers {
         Real64 constexpr Acc(0.0001); // Accuracy of result
         std::string const CalledFrom("SizeEvapFluidCooler");
 
-        int SolFla;               // Flag of solver
-        Real64 UA;                // Calculated UA value [W/C]
-        Real64 OutWaterTempAtUA0; // Water outlet temperature at UA0
-        Real64 OutWaterTempAtUA1; // Water outlet temperature at UA1
+        int SolFla = 0;                    // Flag of solver
+        Real64 UA = 0.0;                   // Calculated UA value [W/C]
+        Real64 OutWaterTempAtUA0 = -999.0; // Water outlet temperature at UA0
+        Real64 OutWaterTempAtUA1 = -999.0; // Water outlet temperature at UA1
 
         Real64 DesEvapFluidCoolerLoad = 0.0; // Design evaporative fluid cooler load [W]
         Real64 tmpDesignWaterFlowRate = this->DesignWaterFlowRate;
         Real64 tmpHighSpeedFanPower = this->HighSpeedFanPower;
         Real64 tmpHighSpeedAirFlowRate = this->HighSpeedAirFlowRate;
 
         int PltSizCondNum = state.dataPlnt->PlantLoop(this->plantLoc.loopNum).PlantSizNum;
+        if (PltSizCondNum > 0) {
+            this->DesignExitWaterTemp = state.dataSize->PlantSizData(PltSizCondNum).ExitTemp;
+            if (this->DesignEnteringWaterTemp == DataSizing::AutoSize) {
+                this->DesignEnteringWaterTemp =
+                    state.dataSize->PlantSizData(PltSizCondNum).ExitTemp + state.dataSize->PlantSizData(PltSizCondNum).DeltaT;
```

Comment:
I might have missed it, but if Design Entering Water Temperature is really "autosizable", then I would expect a corresponding entry reported to the SQL/EIO, no?